iphone/Classes/TiBase.h

```diff
@@ -70,6 +70,11 @@ extern "C" {
   {                            \
     TiLogMessage(__VA_ARGS__); \
   }
+  
+// Hide assertions in prod-builds
+#ifdef NDEBUG
+#define assert(e)   ((void)0)
```

Comment:
Shouldn't assertions automatically be disabled by the compiler if you pass the `-DNDEBUG` flag?

Reply:
Just tried it out, looks okay! PR updated.